### DISCUSSION TITLE
Typo on robot setup steps.

### DISCUSSION THREAD

**Discussion by Gray-Stone** [Yaskawa-Global/YMConnect#22] [General]
When I follow [this guide](https://developer.motoman.com/en/YMConnect/RobotControllerSetup) to setup the robot. I noticed on step2 of setting up `PSEDUO INPUT SIG` the signal number doesn't match what's in the screen shot.

 I assume the signal number should be `87015` given the name of the signal being `CMD REMOTE SEL`

**Comment by ted-miller** [comment]
Hi @Gray-Stone 
You are correct about the signal number. Thank you for pointing out the error. I have corrected the documentation.
